recipes-extended/xen/xen_4.19.bb

```diff
@@ -0,0 +1,15 @@
+FILESEXTRAPATHS:prepend := "${THISDIR}/files:"
```

Comment:
Is there are reason why you are not using recipe provided by meta-xt-common? You commit message does not tell anything about this

Reply:
I'm also don't understand why you decided to remove changes you proposed in one of the already merged prs/

Reply:
@lorc Yes
@oleksiimoisieiev This version doesn't depend on meta-xt-common. It means we don't have to wait until meta-xt-common will move to scarthgap